webui/react/src/pages/F_ExpList/ComparisonView.tsx

```diff
@@ -78,7 +88,7 @@ const ComparisonView: React.FC<Props> = ({
       open={open}
       onChange={onWidthChange}>
       {children}
-      <Pivot destroyInactiveTabPane items={tabs} />
```

Comment:
question: do you know why `destroyInactiveTabPane` was used previously? Removing this doesn't cause any issues? I'm wondering why because `destroyInactiveTabPane` is `false` by default.

Reply:
I previously implemented the `CompareParallelCoordinates` component. There was no strong reasoning for including `destroyInactiveTabPane`. However, now that the compare view is fully implemented it is clear how `destroyInactiveTabPane` which causes each component to unmount, causes a poor `UX` since the data is re-queried for every tab switch. 

Additionally, both the `CompareMetrics` and CompareParallelCoordinates` components were being worked on in parallel. Since the components were being implemented independently it was not clear at the time that the data from `useTrialMetrics` was better suited to be passed into each component.

 The above answers your next question
 
 > Removing this doesn't cause any issues?
 
It would cause issues if the `useTrialMetrics` calls were continuously duplicated in each component. However, now that the metric data is passed into each tab and only the parent component initiates the metric data API calls, the rendering of each tab will be more performant. For example, previously each child would render and load independently based on when the `API` response in each component was received, however now each tab will render completely once the `useTrialMetrics` call completed in the `CompareView` component so switching between tabs will be a smoother experience.